.github/workflows/build.yml

```diff
@@ -286,8 +284,6 @@ jobs:
             ${{ steps.cache-deps.outputs.cache-hit == 'true' && 'NO' || 'YES' }}
       - name: Build
         run: cmake --build build
-      - name: Build tests
-        run: cmake --build build --target tests
       - name: Package
```

Comment:
For MinGW, I understand that it might be a bit difficult to run the tests (should still be doable with `wine`?), but for Windows, it should be easy to just run `ctest`? Sorry, I know, not the scope of the PR :smiley: (reminds me of my PR #6983 which does the same to start `lmms --help` on these systems).

I think without this, we do not test your changes on neither Windows, nor MinGW, so this might introduce a regression there (unless you tested it manually).

Reply:
> For MinGW, I understand that it might be a bit difficult to run the tests (should still be doable with wine?), but for Windows, it should be easy to just run ctest? Sorry, I know, not the scope of the PR 😃 (reminds me of my PR https://github.com/LMMS/lmms/pull/6983 which does the same to start lmms --help on these systems).

@DomClark mentioned that tests dont work well on Windows. This was the reason:
> We don't run them for MinGW because it's a Windows build on a Linux host. We ought to run them for MSVC builds; at least on my machine, there are some that currently fail due to reliance on a particular file system layout, but I'm sure that can be addressed.

But yeah, in the future, we should probably run them on Windows.

Reply:
Just "because it's a Windows build on a Linux host" is not a good reason - You can just use wine, like in my PR. But if it is generally unstable (not sure what Dom meant), it might indeed better to not run them.

Reply:
It might be useful to open an issue so we can remember we can add it in the future once everything has been worked out.

Reply:
Okay, I added tests for Windows, just left MinGW alone because it might over complicate things for no added value.